winui/Masked-TextBox/How-To.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Validation support in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the validation support in the MaskedTextBox (SfMaskedTextBox) control.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# How to restrict the validation in WinUI MaskedTextBox
+
+The MaskedTextBox control provides the ability to validate the input according to the completion of masked input. By default, validation is enabled in the MaskedTextBox control. However, you can disable the validation by assigning `true` to the IsValid property of the ValueChanging event.
+
+{% tabs %}
```

Comment:
@BhaskarSuresh22 `true` to the `IsValid`

Reply:
@Manivannan-E, Done the changes.